OloEngine/src/OloEngine/Renderer/ShaderResourceRegistry.cpp

```
@@ -469,12 +469,14 @@ namespace OloEngine
 
     bool ShaderResourceRegistry::IsStandardUBOBinding(u32 binding, const std::string& name) const
     {
+        OLO_PROFILE_FUNCTION();
+
         // If the name starts with underscore and numbers, it's likely a SPIR-V generated name
         // In this case, we only check the binding point
         if (name.find("_") == 0 && name.length() > 1 && std::isdigit(name[1]))
```

Cppcheck (2.19.0)

[performance] Inefficient usage of string

(stlIfStrFind)